src/codebook/chartMaker/draw.js

```diff
@@ -45,7 +45,7 @@ export function draw(codebook) {
 
     //Define chart.
```

Comment:
👍 This should work.

I wonder if there's a good way to (optionally) pass webcharts a node (or a d3 selection) instead of a css indentifier.

Reply:
Pretty sure that's what's happening, no?